container.go

```diff
@@ -482,6 +482,113 @@ func (c *Client) TopContainer(id string, psArgs string) (TopResult, error) {
 	return result, nil
 }
 
+// NetworkStats describes the data related to the container's memory usage.
+type NetworkStats struct {
+	RXdropped uint64 `json:"rx_dropped"`
+	RXbytes   uint64 `json:"rx_bytes"`
+	RXerrors  uint64 `json:"rx_errors"`
+	TXpackets uint64 `json:"tx_packets"`
+	TXdropped uint64 `json:"tx_dropped"`
+	RXpackets uint64 `json:"rx_packets"`
+	TXerrors  uint64 `json:"tx_errors"`
+	TXbytes   uint64 `json:"tx_bytes"`
```

Comment:
Please use RXDropped, RXBytes, RXErrors, etc. When connecting words, the second words must come with the first letter in uppercase.